src/vcf/validator.cpp

```diff
@@ -20,6 +20,12 @@ namespace ebi
 {
   namespace vcf
   {
+    bool validate(const std::vector<char> &firstLine,
```

Comment:
To make a free function private, it could also be declared in the hpp as static, or in the unnamed namespace. In this case we would be fine with either of them, but "static" forces internal linkage so I would be slightly in favor of that one.

TL;DR link: http://stackoverflow.com/a/11032872

And for more detailed discussions:
http://stackoverflow.com/questions/16864573/is-there-any-diference-between-defining-static-function-inside-anonymous-namespa
http://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions
http://stackoverflow.com/questions/8460327/why-are-anonymous-namespaces-not-a-sufficient-replacement-for-namespace-static